src/modeliteration.js

```diff
@@ -86,7 +86,6 @@ define([
             },
             getExtraDataAttributes: function (mug) {
                 // HACK must happen before <setvalue> and "other" <instance> elements are written
-                prepareForWrite(mug);
```

Comment:
Does the `HACK` comment above this line pertain to this line only or to the whole method? Seems like the former, in which case it should also be removed.